models/edr/dbt_artifacts/dbt_exposures.sql

```diff
@@ -1,6 +1,6 @@
 {{
   config(
-    materialized='table',
+    materialized='incremental',
```

Comment:
Why is there a need to change the materialization?

Reply:
So my reasoning is that the table materialization actually drops the table, where by default the incremental materialization only does [append or merge](https://docs.getdbt.com/docs/build/incremental-models#about-incremental_strategy) - which in this case is equivalent to doing nothing.

So changing to incremental helps to make it atomic, as basically all the logic happens in `upload_artifact_to_table` (which is also why I always truncate now, regardless of whether it runs from a post-hook or the on-run-end hook)